### PR TITLE
gpkg-to-pgdump: Process hexbin geometries too

### LINKED ISSUE #16: Generate the hexbin geometries too in GitHub Actions
This is a follow up to https://github.com/OpenDRR/opendrr-api/issues/85#issuecomment-828618410;
creating this microtask or microissue so that I don't forget.  :sweat_smile:

### PR DESCRIPTION
**Pull request by anthonyfok**
Fixes #16

### CODE REVIEW

**Comment by anthonyfok** [comment]
From the run log at https://github.com/anthonyfok/boundaries/runs/2522886558?check_suite_focus=true

and how https://opendrr.eccp.ca/file/OpenDRR/opendrr-boundaries.dump increases in size from 822.3 MB to 868.0 MB, I believe it is working.  Thanks in advance for reviewing!

File: .github/workflows/gpkg-to-pgdump.yml [hunk 1]
```diff
@@ -88,17 +87,17 @@ jobs:
       - name: Run ogr2ogr to import GeoPackage files into PostGIS
         if: ${{ github.event_name != 'schedule' }}
         run: |
-          ogr2ogr --version
-          for i in *.gpkg; do
-            echo "ogr2ogr: $i"
-            /usr/bin/time ogr2ogr -t_srs "EPSG:4326" -f PostgreSQL PG:"host='$PGHOST' user=postgres dbname='$DB_NAME' password='$PGPASSWORD'" "$i" -lco LAUNDER=NO -nln boundaries.$(basename "$i" .gpkg)
+          stdbuf -oL ogr2ogr --version
+          for i in *.gpkg hexbin_4326/*.gpkg; do
```
Comment (wkhchow) [inline .github/workflows/gpkg-to-pgdump.yml:91]: does it import the other boundaries as well?  Or it doesn't need to?

Comment (anthonyfok) [reply .github/workflows/gpkg-to-pgdump.yml:91]: Hey @wkhchow!

Sorry for sneaking in the "stdbuf" unbuffering commit in this PR too, as it hides what is being changed.

The only relevant change is in the *for* loop where

```sh
for i in *.gpkg
```
was changed to
```sh
for i in *.gpkg hexbin_4326/*.gpkg
```
in order to match not only the *.gpkg in the top-level directory, but also those in the new hexbin_4326/ directory.

You may check the following log to see that both the original geometry files as well as the hexbin ones are processed in the following log:

https://github.com/anthonyfok/boundaries/runs/2522886558?check_suite_focus=true#step:9:1

With the noisy "time" output (used for checking duration and memory usage) removed, you'll see this:

```
ogr2ogr: Geometry_ADAUID.gpkg
ogr2ogr: Geometry_CANADA.gpkg
ogr2ogr: Geometry_CDUID.gpkg
ogr2ogr: Geometry_CSDUID.gpkg
ogr2ogr: Geometry_DAUID.gpkg
ogr2ogr: Geometry_ERUID.gpkg
ogr2ogr: Geometry_FSAUID.gpkg
ogr2ogr: Geometry_PRUID.gpkg
ogr2ogr: Geometry_SAUID.gpkg
ogr2ogr: hexbin_4326/HexGrid_10km.gpkg
ogr2ogr: hexbin_4326/HexGrid_25km.gpkg
ogr2ogr: hexbin_4326/HexGrid_50km.gpkg
ogr2ogr: hexbin_4326/HexGrid_5km.gpkg
ogr2ogr: hexbin_4326/SAUID_HexGrid.gpkg
```

So hopefully that includes all the boundaries that need to be imported.
Please let me know if I missed anything.  :sweat_smile:

Comment (wkhchow) [reply .github/workflows/gpkg-to-pgdump.yml:91]: looks good!

Comment (anthonyfok) [reply .github/workflows/gpkg-to-pgdump.yml:91]: Thanks Will!

**Pull request merged**